app/views/teacher_training_adviser/steps/_previous_teacher_id.html.erb

```diff
@@ -1,5 +1,25 @@
-<% @page_title = "What is your previous teacher reference number?" %>
+<% @page_title = "What is your teacher reference number (TRN)?" %>
 
 <%= f.govuk_fieldset legend: { text: "What is your previous teacher reference number?", tag: "h1" } do %>
   <%= f.govuk_text_field :teacher_id, width: 20 %>
 <% end %>
+
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      How do I find my teacher reference number (TRN)?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    Your TRN is sometime on your payslip, pension statement or teacher training records.<br>
+    Alternatively, you can find a lost TRN<br><br>
+    You'll be asked for your:
```

Comment:
@MaxineCl could you please confirm if having **Alternatively, you can [find a lost TRN](https://find-a-lost-trn.education.gov.uk/start). You'll be asked for your:** in a single line makes sense? If yes, John, could you please make the change?